package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@morgul/knex-db2",
-  "version": "0.4.0",
```

Comment:
Don't bump versions in pull requests like this. It's up to me to decide to release a new version.

Reply:
Should be fixed in latest commits.